public/css/admin.css

```diff
@@ -12,6 +12,30 @@
   user-select:none
 }
 
+.select-wrapper span.caret {
```

Comment:
please change this to 4 character spacing to be consistent with the rest of the file

Reply:
fixed